### ISSUE TITLE
Add to the documentation that it's currently support on Node <=10

### ISSUE THREAD

**Issue by tomafc330** [kinecosystem/kin-sdk-node#48]
The latest version of NodeJS (12.x.x) doesn't support some of the functions used by the library. In particular:
```
    "@kinecosystem/kin-base": {
      "version": "0.20.4",
      "resolved": "https://registry.npmjs.org/@kinecosystem/kin-base/-/kin-base-0.20.4.tgz",
      "integrity": "sha512-Hb5umvYV7bqAEnAawJDaSwNygx8I4tQHVHAuUWdbOESw2Y2XzhA3Csl81xvEvrPsghvkYjwsZA9F/YuKqEIWPw==",
      "requires": {
        "base32.js": "0.1.0",
        "bignumber.js": "5.0.0",
        "crc": "3.8.0",
        "ed25519": "0.0.4",
        "js-xdr": "1.1.2",
        "lodash": "4.17.14",
        "sha.js": "2.4.11",
        "tweetnacl": "1.0.1"
      },
```

ed25519 is the culprit, it's an encryption library but it's pretty outdated and uses removed APIs in Node version 12. 

I will create a PR to update the README to add a note :)

**Comment by tomafc330** [comment]
Updated the README: 

https://github.com/kinecosystem/kin-sdk-node/pull/49

**Comment by beeman** [comment]
I have noticed these issues using node v12 and I think the fix here should be something that works with that version. Node.js v10 will go into maintenance mode on April 2020.

There seems to be a native JavaScript implementation but they say it's much slower. Not sure if there are other options to get rid of this dependency

**Comment by chaseeb** [comment]
Thanks @beeman I'll make sure and stay on top of this change as we'll need to make sure it's taken care of. Are you saying that there is not an updated version of ed25519?

**Comment by beeman** [comment]
@chaseeb that's correct. The last version of that package is from 2015 and it seems like it's abandoned.

If you check this warning here you see a fallback is used, but it's recommended against for production use:
https://github.com/stellar/js-stellar-sdk/blob/master/README.md#stellar-sdk-vs-stellar-base

Maybe this can be used on Node 12 and up: https://github.com/spacemeshos/ed25519-WASM

**Comment by tomafc330** [comment]
Just chiming in here -- looked at the stellar commits, found this gem:

https://github.com/stellar/js-stellar-base/commit/bd6c26845d21329e08081e385fd3656f103ff3d4

**Comment by chaseeb** [comment]
Nice fine, Tommy!  Thanks for looking into this =)

**Comment by tomafc330** [comment]
No problem -- glad to help :) 

Can probably close my PR once that change has been implemented

**Comment by beeman** [comment]
Nice one @tommytcchan, seems like a great fix 💯
